operator/pkg/gateway-api/conformance_test.go

```diff
@@ -32,6 +42,38 @@ import (
 //		--debug -test.run "TestConformance/HTTPRouteDisallowedKind"
 func TestConformance(t *testing.T) {
 	testutils.GatewayAPIConformanceTest(t)
-
-	conformance.RunConformance(t)
+	var skipTests []string
+	options := conformance.DefaultOptions(t)
+	var usableNetworkAddresses []v1beta1.GatewayAddress
+	var unusableNetworkAddresses []v1beta1.GatewayAddress
+	if os.Getenv(usableNetworkAddressesEnv) == "" {
+		t.Logf("Set %s to run this test", features.SupportGatewayStaticAddresses)
+		skipTests = append(skipTests, string(features.SupportGatewayStaticAddresses))
+	} else {
+		var addressType = v1.IPAddressType
+		usableNetworkAddressesStr := os.Getenv(usableNetworkAddressesEnv)
```

Comment:
Nit: This call to `os.Getenv()` could be before the `if`, and then there would be only one call rather than two. Same for `unusableNetworkAddressesEnv`.

Not a super big deal here, but if this call was expensive or in a hot path, it would matter more.